api/recipient/recipientModel.js

```diff
@@ -1,5 +1,22 @@
 const knex = require('../../data/db-config');
 
+const findAll = () => {
+  return knex('recipients as r')
+    .leftJoin('households as h', 'r.household_id', 'h.household_id')
```

Comment:
Nice job joining all of these tables. Thank you for doing this to give extra functionality on the frontend.